build.sbt

```diff
@@ -26,7 +26,7 @@ ThisBuild / crossScalaVersions := Seq(Scala213, "3.2.2")
 ThisBuild / scalaVersion := Scala213 // the default Scala
 
 val CatsVersion = "2.9.0"
-val CatsEffectVersion = "3.5.0"
+val CatsEffectVersion = "3.5.0-30-0a69caf"
```

Comment:
Just pushed https://github.com/typelevel/cats-effect/commit/27750647101d150d7006b7917315e80a88759c17 which covers all of the `delay` and `blocking` stages in the interpreter.